src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Suppressions.LibraryBuild.xml

```diff
@@ -8,5 +8,26 @@
       <property name="Target">M:System.StartupHookProvider.ProcessStartupHooks()</property>
       <property name="Justification">This warning is left in the product so developers get an ILLink warning when trimming an app with System.StartupHookProvider.IsSupported=true.</property>
     </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2026</argument>
+      <property name="Scope">member</property>
+      <property name="Target">M:Internal.Runtime.InteropServices.ComponentActivator.LoadAssemblyAndGetFunctionPointer(System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr)</property>
+      <property name="Justification">This warning is left in the product so developers get an ILLink warning when trimming an app with Internal.Runtime.InteropServices.ComponentActivator.IsSupported=true.</property>
+    </attribute>
+    <attribute fullname="System.Diagnostics.CodeAnalysis.UnconditionalSuppressMessageAttribute">
+      <argument>ILLink</argument>
+      <argument>IL2026</argument>
+      <property name="Scope">member</property>
+      <property name="Target">M:Internal.Runtime.InteropServices.ComponentActivator.GetFunctionPointer(System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr,System.IntPtr)</property>
+      <property name="Justification">This warning is left in the product so developers get an ILLink warning when trimming an app with Internal.Runtime.InteropServices.ComponentActivator.IsSupported=true.</property>
```

Comment:
Why are we leaving 2 warnings in the product for the same switch?

Reply:
This is a bit tricky - there are two different entry points for the same feature. The feature switch should enable both, but from analysis perspective they are two separate things, and thus should be annotated separately.

Reply:
I get that we have 2 places where we call from the native host into managed code. But does a developer really care how many places we call internally?

The point of these "LibraryBuild" suppressions is that when a dev trims their app with these "problematic" settings, they get an actionable message telling them what about their settings is problematic, and what they can do to address it.

Does giving them essentially the same warning message twice add value?

Reply:
It doesn't - but I don't want to add the attribute only on one of them (at random) - that doesn't feel right. We could put it on an internal method which called by both (basically just for this purpose) - it's also not very nice... don't know.

Reply:
> I don't want to add the attribute only on one of them (at random) 

That's essentially what we did for ResourceManager:

https://github.com/dotnet/runtime/blob/b491d634bb793447bf0dbe87eb70a03c4e75bbe1/src/libraries/System.Private.CoreLib/src/System/Resources/ResourceReader.Core.cs#L48-L52